README.md

```diff
@@ -486,6 +486,152 @@ rescue Stormpath::Error => e
 end
 ```
 
+### Authentication Against a SAML Directory
+
+SAML is an XML-based standard for exchanging authentication and authorization data between security domains. 
+Stormpath enables you to allow customers to log-in by authenticating with an external SAML Identity Provider.
+
+#### Stormpath as a Service Provider
+
+The specific use case that Stormpath supports is user-initiated single sign-on. In this scenario, a user requests 
+a protected resource (e.g. your application). Your application, with the help of Stormpath, then confirms the users 
+identity in order to determine whether they are able to access the resource. In SAML terminology, the user is the User 
+Agent, your application (along with Stormpath) is the Service Provider, and the third-party SAML authentication site 
+is the Identity Provider or IdP.
+
+The broad strokes of the process are as follows:
+
+* User Agent requests access from Service Provider
+* Service Provider responds with redirect to Identity Provider
+* Identity Provider authenticates the user
+* Identity provider redirects user back to Service Provider along with SAML assertions.
+* Service Provider receives SAML assertions and either creates or retrieves Account information
+
+#### Configuring Stormpath as a Service Provider
+
+Configuration is stored in the Directory's Provider resource. Both of these resources must also be linked with an 
```

Comment:
When you say "Both if these resources", do you mean the application and the directory? In any case, please add more information to be more explicit.